app/uk/gov/hmrc/apiplatform/modules/submissions/repositories/QuestionnaireDAO.scala

```diff
@@ -349,7 +349,8 @@ object QuestionnaireDAO {
         QuestionId("10249171-e87a-498e-8239-a417af29e2ff"),
         Wording("Can your customers access their data?"),
```

Comment:
Wording now looks to be 'Can customers access their data?'

Reply:
Well spotted - fixed